FORMULA

```diff
@@ -0,0 +1,8 @@
+name: napalm-logging
+os: Junos, IOS-XR, EOS
+os_family: RedHat, Debian, Suse, Gentoo
+version: 201709
+release: 1
+minimum_version: 2017.9.0
```

Comment:
This refers to the minimum Salt version to be have to be able to execute this state.